packages/components/src/Content/Content.tsx

```diff
@@ -14,7 +15,20 @@ interface ContentProps {
 }
 
 export function Content({ children, spacing = "base" }: ContentProps) {
-  const className = classnames(styles.padded, spacings[spacing]);
+  const [ref, result] = useResizeObserver();
```

Comment:
I think for the documentation we should write this as 

```js
const [ref, {width, height}] =  useResizeObserver()
```

Reply:
Noted. Although, this is going to get pulled out of the PR when it ships.
Thoughts on where we should document the hook useage?

Reply:
I think that docz should have a page/section for hooks.

Reply:
Good call - I added the hooks package to the docz site, so that it pulls the Readme. I'll add this as an example in that Readme.